scene/gui/rich_text_label.cpp

```diff
@@ -2615,6 +2615,7 @@ void RichTextLabel::pop() {
 // Creates a new line without adding an ItemNewline to the previous line.
 // Useful when wanting to calling remove_line and add a new line immediately after.
 void RichTextLabel::increment_line_count() {
+	_validate_line_caches(main);
```

Comment:
I think this was a new function added since I first wrote this PR? I definitely would have noticed #48450 in testing when I originally wrote it... Anyway, this seems to resolve the issue at least.

Reply:
You seem to have referenced an unrelated PR here.

Reply:
Hmm.. was meant to be #48443